frontend/src/APIClients/AdminAPIClient.ts

```diff
@@ -0,0 +1,17 @@
+import { BEARER_TOKEN } from "../constants/AuthConstants";
+import baseAPIClient from "./BaseAPIClient";
+
+const getWaiver = async () => {
+  try {
+    const { data } = await baseAPIClient.get(`/admin/waiver`, {
+      headers: { Authorization: BEARER_TOKEN },
+    });
+    return data;
+  } catch (error) {
+    return {};
```

Comment:
Can you return the following instead:
```
catch(error) {
    return error as WaiverResponse;
}
```
I am not sure how the parsing is working but looking at other API clients, we return the error casted as the return type when we get an error so it should work